src/components/Form/index.tsx

```diff
@@ -199,9 +257,11 @@ export const Form = ({
               error={Boolean(errors.claim)}
               helperText={errors.claim?.message}
             >
-              {inputOptions.claim.map((i: string) => (
+              {inputOptions.claim.map((i: string, index: number) => (
```

Comment:
`i` as a variable is widely used to denote `iteration` in programming. Therefore, using `i` to mean something else is very confusing. In this case the ideal name would be `claim` or `claimText` etc.